distribution/adapters/js-adapter-npm-zip/src/main/resources/package.json

```diff
@@ -20,5 +20,9 @@
     "oauth",
     "oauth2",
     "authentication"
-  ]
+  ],
+  "dependencies": {
```

Comment:
Why add these dependencies when they are imported? The dependencies here won't actually end up being used right?

Reply:
Never mind, read your description.